DEPENDENCIES

```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/1.4.0, Apache-2.0 AND CC-BY-4.0 AND OFL-1.1, approved, #9102
+npm/npmjs/-/cx-portal-shared-components/1.5.0, Apache-2.0 AND (Apache-2.0 AND LicenseRef-scancode-unknown) AND CC-BY-4.0 AND OFL-1.1, restricted, #10235
```

Comment:
@almadigabor  ticket is created in the eclipse side already. Won't this be enough to go ahead for releasing? We have done like this before.